CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ https://github.com/bitcrowd/sshkit.ex/compare/v0.0.3...HEAD
 
 ### Fixes:
 
-* Put fixes here
+* Context properly handles the case where env is set to an empty map
```

Comment:
We just merged a PR prep-ing the next release which added a bunch of "Fix" notes. A rebase onto our current master might be useful to verify this message ends up in the correct spot 😀 (maybe I'm underestimating git)

Reply:
will do!

Reply:
Shall i put these in the `0.1.0` Fix section of in the `master` Fix section?

Reply:
You can put it in the `0.1.0` section 🤘

Reply:
We haven't published that version yet, but are just preparing it.